docs/api/ShallowWrapper/prop.md

```diff
@@ -1,22 +1,44 @@
 # `.prop(key) => Any`
 
-Returns the prop value for the node of the current wrapper with the provided key.
+Returns the prop value for the root node of the wrapper with the provided key.
+`.prop(key)` can only be called on a wrapper of a single node.
 
-NOTE: can only be called on a wrapper of a single node.
+NOTE: When called on a shallow wrapper, `.prop(key)` will only return values for props that exist for the parent element of the component, not the whole React component. To return the props for the
```

Comment:
this might be clearer stated as `… will return values for props on the root node that the component *renders*, not the component itself. To return the props…`